<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWorkExperienceWindow</class>
 <widget class="QMainWindow" name="AddWorkExperienceWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Work Experience</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Add Work Experience</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelYears">
      <property name="text">
       <string>Years</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputYears"/>
    </item>
    <item>
     <widget class="QLabel" name="labelCompanyName">
      <property name="text">
       <string>Company Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputCompanyName"/>
    </item>
    <item>
     <widget class="QLabel" name="labelPosition">
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputPosition"/>
    </item>
    <item>
     <widget class="QLabel" name="labelExperience">
      <property name="text">
       <string>Experience</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="inputExperience"/>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="columnCount">
       <number>4</number>
      </property>
      <property name="horizontalHeaderLabels">
       <stringlist>
        <string>Years</string>
        <string>Company Name</string>
        <string>Position</string>
        <string>Experience</string>
       </stringlist>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
